### DISCUSSION TITLE
为什么我下载不了 T_T

### DISCUSSION THREAD

**Discussion by presidon05** [superpung/tju-expense#7] [Q&A]
![Screenshot 2024-12-29 201956](https://github.com/user-attachments/assets/770fcbc9-b50a-4fb1-aa95-d7c0af1da1bf)

**Comment by superpung** [comment]
可能需要你开启某些权限，或者换个网络？我这边现在可以正常下载的。有没有更多信息呢？

**Comment by superpung** [comment]
https://github.com/superpung/tju-expense/releases/latest/download/tju-expense-windows.exe
这是下载链接，可以尝试换个浏览器或下载工具试试。

**Reply by presidon05** [reply]
换了个浏览器，可以了，可能刚那个自带防毒。谢谢！
